### ISSUE TITLE
Stable version of busted broken

### ISSUE THREAD

**Issue by nilnor** [Olivine-Labs/busted#391]
The last stable version of busted (1.11.1-1) appears to be broken, apparently with the recent 2.4 release of lua_cliargs:

$ busted
/usr/local/bin/busted_bootstrap:49: attempt to call a nil value

Downgrading to lua_cliargs 2.3 solves the issue.

**Comment by kikito** [comment]
I just created this: https://github.com/kikito/busted-stable

**Comment by nilnor** [comment]
A laudable effort @kikito, but unfortunately it didn't work for me when I tried it, since the latest lua_cliargs was pulled down as well (presumably as the dependency of a dependency) and it still got picked up.

I share your apparent frustration however. There I was, happily busting along, when all of a sudden an incomplete, backwards incompatible -RC version became the default  luarocks install. And then even fixing the version wasn't enough and it started failing silently (due to a wrong exit code). 

@ajacksified @DorianGray: Any feedback on this?

**Comment by mpeterv** [comment]
Perhaps it would be better to upload a new revision of 1.11.1 rockspec with pinned dependency on lua_cliargs?

**Comment by o-lim** [comment]
I created a branch that fixes this issue: https://github.com/o-lim/busted/tree/fix-busted-stable.  Just do
```
git clone -b fix-busted-stable --single-branch https://github.com/o-lim/busted
```

I tried to submit a pull request for this fix, but github would not let me create a PR against a tag.

**Comment by ajacksified** [comment]
Released v1.11.1-2 that fixes cliargs - thanks for the fix, @o-lim .
